mlir/lib/Conversion/MIGraphXToTosa/MIGraphXToTosa.cpp

```diff
@@ -965,7 +969,8 @@ LogicalResult DeQuantizeLinearConverter::matchAndRewrite(
         rewriter, loc, outputType, upcastInput, upcastBias);
   }
   tosa::MulOp scaled =
-      rock::tosa::getMulOp(rewriter, loc, shifted, scale, outputType);
+      rock::tosa::getMulOp(rewriter, loc, shifted, scale,
+                           getTypeConverter()->convertType(outputType));
```

Comment:
```suggestion
                           outputType);
```
It is already converted.